quality_check/11-rnaseq_bias.Rmd

```diff
@@ -0,0 +1,294 @@
+only the RNA-seq samples
+
+```{r}
+pc_file <- file.path("results", "larger_compendium_PC1and2.tsv")
+seq_df <- readr::read_tsv(pc_file) %>%
+  dplyr::filter(compendium_technology_labels != "MICROARRAY")
+```
+
+We're going to somewhat arbitrarily divide samples into two groups on the basis
+of their values for PC1.
+
+```{r}
+seq_df <- seq_df %>%
+  dplyr::mutate(Group = dplyr::case_when(
+    PC1 > 0 ~ "positive",
+    PC1 < 0 ~ "negative"
+  ))
+```
+
+How's this look?
+
+```{r}
+seq_df %>%
+  ggplot(aes(x = PC1, y = PC2, colour = Group)) +
+  geom_point() +
+  theme_bw()
+```
+
+This is probably close enough for what we're after here.
+
+### Random subset of samples from each group
+
+```{r}
+positive_samples <- sample(seq_df %>% 
+                             dplyr::filter(Group == "positive") %>% 
+                             dplyr::pull(Sample), 
+                           50)
+```
+
+```{r}
+negative_samples <- sample(seq_df %>% 
+                             dplyr::filter(Group == "negative") %>% 
+                             dplyr::pull(Sample), 
+                           50)
+```
+
+We took a look at a handful of samples and couldn't discern a pattern of poly-A
+enrichment vs. rRNA depletion.
+So we decided to dig a bit deeper into metadata from the `salmon quant` runs 
+themselves. 
+(See all the custom functions above.)
+
+### Extract (`salmon quant`) metadata about the two groups
+
+Extract the relevant metadata for the random subset of positive samples
+
+```{r}
+positive_metadata_df <- get_metadata_df(positive_samples)
+positive_metadata_df %>%
+  dplyr::arrange(experiment)
+```
+
+Extract the relevant metadata for the random subset of negative samples
+
+```{r}
+negative_metadata_df <- get_metadata_df(negative_samples)
+negative_metadata_df %>%
+  dplyr::arrange(experiment)
+```
+
+#### Notes on features of two groups
+
+* The positive group contains a mix of library layouts, instruments, and short
+and long read lengths.
+* The negative group is entirely paired-end samples with short read lengths and 
+generally it looks like there are lower % reads mapped. _However_ only a few 
+experiments appear to be represented in this group.
+
+### What experiments are represented in the "negative" group?
+
+Do all the negative samples come from very large experiments?
+Because `ERP008771` and `ERP006132` each have over 2000 samples.
+
+```{r}
+# to save on compute time, we'll sort the sample accession codes so runs from
+# the same experiment should be next to each other and then examine every
+# 25th sample
+all_negative_samples <- sort(seq_df %>% 
+  dplyr::filter(Group == "negative") %>% 
+  dplyr::pull(Sample))
+indices <- seq(1, length(all_negative_samples), 25)
+negative_experiment_accession_codes <- 
+  sapply(all_negative_samples[indices], get_sample_project_accession)
+# display in a readable format
+data.frame(sort(unique(negative_experiment_accession_codes)))
+```
+
+These are all from the Wellcome Trust Sanger Institute.
```

Comment:
Need extra empty line.

Reply:
Also I feel like the story you were making here ended so abruptly. I think I know what your conclusion is, but can you just put a few sentences at the end to summarize your conclusions?